src/core/interpreter.cc

```diff
@@ -323,6 +324,23 @@ int RedisSha1Command(lua_State* lua) {
   return 1;
 }
 
+int RedisIHashCommand(lua_State* lua) {
```

Comment:
what's this? part of this PR?

Reply:
ah, it's your idea to  support tests in lua?

Reply:
Oh damn, git stash gone wrong, will remove it 😬